packages/std/src/binary.rs

```diff
@@ -116,8 +116,8 @@ macro_rules! implement_from_for_fixed_length_arrays {
             // Owned
             impl From<[u8; $N]> for Binary {
                 fn from(source: [u8; $N]) -> Self {
-                    // Implementation available for $N <= 32.
-                    // Requires https://caniuse.rs/features/vec_from_array, avaiable since Rust 1.44.0.
+                    // Implementation available for $N <= 64.
+                    // Requires https://caniuse.rs/features/vec_from_array, available since Rust 1.44.0.
```

Comment:
```suggestion
                    // Requires https://caniuse.rs/features/vec_from_array available since Rust 1.44.0
                    // as well as "Traits on larger arrays" (https://blog.rust-lang.org/2020/10/08/Rust-1.47.html#traits-on-larger-arrays)
                    // available since Rust 1.47.0
```